javascript/src/html_processor.ts

```diff
@@ -230,17 +230,20 @@ class NodeOrText {
     this.nodeOrText = nodeOrText;
   }
 
-  get isString(): boolean {
+  private isString(value: Text | string): value is string {
     return typeof this.nodeOrText === 'string';
```

Comment:
Interesting, I didn't know TS can do this.

In this new code, the `value` isn't used. If your intention is to make this function for any inputs, I think a) it should be a static function, and b) it should check `value` instead of `this.nodeOrText`.

If this function always whether `this.nodeOrText` is string or not, well, I'm not sure if we can do `this.nodeOrText is string` as the type specifier?

Reply:
Thanks for reviewing! Yes, my intention was to accept any input. Fixed to make the function a static function.